README.md

```diff
@@ -12,7 +12,19 @@
 
 
 ```v
-ui.window(
+import ui
+
+
+struct App {
+mut:
+	window     &ui.Window = unsafe { nil }
```

Comment:
It seems `v fmt` was omitted (e.g. there are tabs instead of spaces). Could you (re-)run it?